src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="http:localhost:8080/api" method="post">
+
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1" max="100"
+          >
+        </label>
+
+        <div class="form-field">
+          <label for="data">Full date of birth:</label>
+          <input
+            type="data"
+            id="data"
+            name="data"
+            value="dd. mm. yyyy">
```

Comment:
typo, date, not data
```suggestion
          <input
            type="date"
            id="date"
            name="date"
          >
```

Reply:
![image](https://user-images.githubusercontent.com/99331793/187875569-7572f68b-5ead-453d-90a6-2c2321fedc6f.png)